.github/pull_request_template.md.template.py

```diff
@@ -588,14 +588,36 @@ def main():
                 'type': 'cli',
                 'content': 'Pushed merge commit to GitHub'
             },
+            *iif(t in (T.default, T.gitlab), [
+                {
+                    'type': 'h2',
+                    'content': 'Operator (chain shortening)'
+                },
+                {
+                    'type': 'cli',
+                    'content': 'Changed the target branch of the blocked PR to `develop`',
+                    'alt': 'or this PR is not labeled `base`'
```

Comment:
The `alt` and `type` keys have the same value in these four dicts. Please condense this with a comprehension over a list of the values for the `content` key.